app/models/mixins/relationship_mixin.rb

```diff
@@ -274,12 +330,20 @@ def child_count(*args)
 
   # Returns true if the record has any children, false otherwise
   def has_children?(*_args)
```

Comment:
ooh, maybe THIS is the `cant find _args`

NOTE:
when the arguments are different from ancestry, pass in the ancestry params, not these.